show/main.py

```diff
@@ -966,6 +966,25 @@ def counters(ctx, verbose, period, printall):
 
         run_command(cmd, display_cmd=verbose)
 
+# 'counters' subcommand ("show interfaces counters errors")
```

Comment:
Should read `# 'errors' subcommand ("show interfaces counters errors")`

Reply:
fixed.